urlx/parse.go

```diff
@@ -17,7 +58,7 @@ func ParseOrPanic(in string) *url.URL {
 
 // ParseOrFatal parses a url or fatals.
 func ParseOrFatal(l *logrusx.Logger, in string) *url.URL {
-	out, err := url.Parse(in)
+	out, err := Parse(in)
```

Comment:
Let's revert this to not break anything in downstream code.

Reply:
I understand your concern, what do you think about adding an additional function so that we can have both functions?
`ParseOrFatal2()`?